docs/src/physics/hydrostatic_free_surface_model.md

```diff
@@ -26,7 +28,7 @@ condition at the surface
 
 ## The momentum conservation equation
 
-The equations governing the conservation of momentum in a rotating fluid, including buoyancy
+The equations governing the conservation of horizontal momentum in a rotating fluid, including buoyancy
```

Comment:
These are conservation of the 3d momentum, no? (where conservation of vertical momentum is simplified by neglecting vertical acceleration as well as non-linear terms and results in the hydrostatic relation)

Reply:
ok, fair!

Reply:
[ff3af9b](https://github.com/CliMA/Oceananigans.jl/pull/3150/commits/ff3af9b6ba2582bc0a9ad634b3c097e7ed6daf26) resolved this